### PR TITLE
[docs] formatting and style tweaks for various pages

### PR DESCRIPTION
**Pull request by Simek**
# Why

Update formatting and phrasing on few pages.

# How

The following change have been made:
* emphasize and move certain callouts
* convert shortcuts list on Expo CLI page to table for readability
* add icons to the few `BoxLink`s on What is Expo page
* rephrase and reformat of requirements on Installation page

Additionally, I have made a change in `BoxLink` to support properly logo icons.

# Test Plan

The changes have bben tested by running docs website locally.

# Preview

Screenshot 2022-10-28 at 16 19 51
Screenshot 2022-10-28 at 16 26 05
Screenshot 2022-10-28 at 16 20 24

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/get-started/installation.mdx [hunk 1]
```diff
@@ -4,42 +4,48 @@ title: Installation
...
-[Expo CLI](/workflow/expo-cli) is a command-line tool that is the primary interface between a developer and other Expo tools. You are going to use it for different tasks in the development life cycle of your project such as serving the project in development, viewing logs, opening the app on an emulator or a physical device, and so on.
+[Expo CLI](/workflow/expo-cli) is a command-line tool that is the primary interface between a developer and other Expo tools.
+You are going to use it for different tasks in the development life cycle of your project such as serving the project in development, viewing logs, opening the app on an emulator or a physical device, and so on.
 
 ### Requirements
 
 To use Expo CLI, you need to have the following tools installed on your developer machine:
 
 - [Node.js LTS release](https://nodejs.org/en/)
+  > Only Node.js LTS releases (even-numbered) are recommended. As Node.js [officially states](https://nodejs.org/en/about/releases/), "Production applications should only use Active LTS or Maintenance LTS releases.".
```
Comment (amandeepmittal) [inline docs/pages/get-started/installation.mdx:23]: Nesting the callout here doesn't look good IMO. The spacing looks odd and also there is no margin between the bullet point and the callout.

CleanShot 2022-10-29 at 13 31 22@2x

Also, there is no need for double periods at the end of the callout: `Maintenance LTS releases.".` can be `Maintenance LTS releases".`

File: docs/pages/get-started/installation.mdx [hunk 2]
```diff
@@ -4,42 +4,48 @@ title: Installation
...
+- For macOS or Linux users: [Watchman](https://facebook.github.io/watchman/docs/install#buildinstall)
 
 ### Recommended tools
 
-- [VS Code Editor](https://code.visualstudio.com/download)
-  - [VS Code Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for **app.json** debugging and autocomplete
 - [Yarn](https://classic.yarnpkg.com/en/docs/install)
-- Windows users: [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows), Bash via WSL, or the VS Code terminal
+- [VS Code Editor](https://code.visualstudio.com/download)
+  - [VS Code Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for **app.json** or **app.config.js** debugging and autocomplete.
```
Comment (amandeepmittal) [inline docs/pages/get-started/installation.mdx:31]:
```suggestion
  - [VS Code Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for Expo config debugging and autocomplete.
```

I think we should start referencing **app.json** or **app.config.js** as Expo config (and if required, add a link to Expo config to this page: https://docs.expo.dev/workflow/configuration/)

File: docs/pages/get-started/installation.mdx [hunk 3]
```diff
@@ -49,16 +55,19 @@ If you already have an Expo account, you can log in to it by running the command
...
+Open the Expo Go app after it has finished installing. If you have created an account with Expo CLI, you can sign in by clicking the "Login" button in the top header on the "Home" tab.
+Signing in will make it easier for you to open projects in the Expo Go app while developing them &mdash; they will appear automatically under the "Projects" section on the Home tab of the app.
 
-> It's often useful to be able to run your app directly on your computer instead of on a separate physical device. If you would like to set this up, you can learn more about [installing an Android Emulator](/workflow/android-studio-emulator) and [installing the iOS Simulator (macOS only)](/workflow/ios-simulator) .
+> It's often useful to be able to run your app directly on your computer instead of on a separate physical device.
+> If you would like to set this up, you can learn more about [installing an Android Emulator](/workflow/android-studio-emulator) and [installing the iOS Simulator (macOS only)](/workflow/ios-simulator) .
```
Comment (amandeepmittal) [inline docs/pages/get-started/installation.mdx:70]:
```suggestion
> If you would like to set this up, you can learn more about [installing an Android Emulator](/workflow/android-studio-emulator) and [installing the iOS Simulator (macOS only)](/workflow/ios-simulator).
```

Unrelated to the changes in this PR: there is an extra whitespace above.

File: docs/pages/get-started/installation.mdx (hunk 1 above)
Comment (Simek) [reply docs/pages/get-started/installation.mdx:23]: What do you think about ditching the callout and leaving the content as a part of the bullet (like description)?

Comment (amandeepmittal) [reply docs/pages/get-started/installation.mdx:23]: Yeah, i think making it part of the bullet sounds better.

**Review by amandeepmittal — APPROVED** [review]

File: docs/pages/get-started/installation.mdx [hunk 4]
```diff
@@ -4,42 +4,48 @@ title: Installation
...
-- [Watchman](https://facebook.github.io/watchman/docs/install#buildinstall) required only for macOS or Linux users
-
-> Only Node.js LTS releases (even-numbered) are recommended. As Node.js [officially states](https://nodejs.org/en/about/releases/), "Production applications should only use Active LTS or Maintenance LTS releases."
+- For macOS or Linux users: [Watchman](https://facebook.github.io/watchman/docs/install#buildinstall)
 
 ### Recommended tools
 
-- [VS Code Editor](https://code.visualstudio.com/download)
-  - [VS Code Expo Extension](https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo) for **app.json** debugging and autocomplete
 - [Yarn](https://classic.yarnpkg.com/en/docs/install)
```
[the remark this answers is not in the document]
Comment (Simek) [reply docs/pages/get-started/installation.mdx:29]: As discussed, we will made a follow-up PR for the Yarn requirement if needed.

**Pull request merged**